### ISSUE TITLE
mamba v0.21.1 crashes creating environment with "intake" channel but v0.21.0 does not

### ISSUE THREAD

**Issue by zaneselvans** [mamba-org/mamba#1489]
I just updated to mamba 0.21.1 and am suddenly unable to build an environment that works fine with mamba 0.21.0.

After getting the error below, I downgraded to mamba 0.21.0 and ran the same command, and it worked fine.  If I remove the environment, upgrade mamba, and run the command again, I get the same error.

The command and error output:

```
mamba env create --force --quiet --name pudl-dev --file devtools/environment.yml

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/zane/miniconda3/lib/python3.8/site-packages/conda/exceptions.py", line 1080, in __call__
        return func(*args, **kwargs)
      File "/home/zane/miniconda3/lib/python3.8/site-packages/conda_env/cli/main.py", line 80, in do_call
        exit_code = getattr(module, func_name)(args, parser)
      File "/home/zane/miniconda3/lib/python3.8/site-packages/conda_env/cli/main_create.py", line 141, in execute
        result[installer_type] = installer.install(prefix, pkg_specs, args, env)
      File "/home/zane/miniconda3/lib/python3.8/site-packages/mamba/mamba_env.py", line 53, in mamba_install
        index = load_channels(pool, channel_urls, repos, prepend=False)
      File "/home/zane/miniconda3/lib/python3.8/site-packages/mamba/utils.py", line 171, in load_channels
        repo = subdir.create_repo(pool)
    RuntimeError: [json.exception.type_error.302] type must be string, but is null

`$ /home/zane/miniconda3/condabin/mamba create --force --quiet --name pudl-dev --file devtools/environment.yml`

  environment variables:
                 CIO_TEST=<not set>
  CONDA_AUTO_UPDATE_CONDA=false
                CONDA_EXE=/home/zane/miniconda3/bin/conda
         CONDA_PYTHON_EXE=/home/zane/miniconda3/bin/python
               CONDA_ROOT=/home/zane/miniconda3
              CONDA_SHLVL=0
           CURL_CA_BUNDLE=<not set>
                  MANPATH=/usr/local/man:
                     PATH=/home/zane/miniconda3/condabin:/usr/local/bin:/usr/local/sbin:/usr/sbi
                          n:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
         PYTHONBREAKPOINT=ipdb.set_trace
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : None
            shell level : 0
       user config file : /home/zane/.condarc
 populated config files : /home/zane/.condarc
          conda version : 4.11.0
    conda-build version : not installed
         python version : 3.8.8.final.0
       virtual packages : __linux=5.13.0=0
                          __glibc=2.34=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/zane/miniconda3  (writable)
      conda av data dir : /home/zane/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/zane/miniconda3/pkgs
                          /home/zane/.conda/pkgs
       envs directories : /home/zane/miniconda3/envs
                          /home/zane/.conda/envs
               platform : linux-64
             user-agent : conda/4.11.0 requests/2.26.0 CPython/3.8.8 Linux/5.13.0-28-generic ubuntu/21.10 glibc/2.34
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.
```

The `environment.yml` file defining the environment:

```yaml
name: pudl-dev
channels:
  - conda-forge
  - defaults
  - intake
dependencies:
  - pip~=21.0
  - python>=3.8,<3.11
```

The problem seems to be related to the [intake channel](https://anaconda.org/intake/repo) provided by Anaconda.  If I remove that channel, then both mamba 0.21.0 and 0.21.1 are able to build the environment.

**Comment by wolfv** [comment]
Thanks! Fixed in #1490

**Comment by wolfv** [comment]
This is fixed in mamba 0.21.2

Cheers!
